indico/modules/events/contributions/controllers/management.py

```diff
@@ -143,8 +144,12 @@ def _process(self):
         contrib_report_args = self.reporter.get_contrib_report_kwargs()
         selected_entry = request.args.get('selected')
         selected_entry = int(selected_entry) if selected_entry else None
+        contribs = contrib_report_args['contribs']
+        total_duration = (sum([c.duration for c in contribs], timedelta()),
```

Comment:
`()` does the job too.. you just cannot have an unparenthesized generator expression in a function call that has additional argument

`foo((x for x in y), bar)`